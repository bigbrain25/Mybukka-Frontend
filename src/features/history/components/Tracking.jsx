import React, { Fragment } from 'react';
import PropTypes from 'prop-types';
import { connect } from 'react-redux';
import shortId from 'shortid';
import TrackingHeader from './TrackingHeader';
import Timeline from '../common/Timeline';
import trackingDisplayClose from '../actionCreators/trackingDisplayClose';

import './tracking.scss';

const Tracking = ({ show, status, closeTrackingDropdown }) => {
  const colorObj = {
    [status]: 'timeline-active',
  };

  const trackingStatus = {
    delivered: {
      text: 'order dispatched',
      icon: '',
      time: '5/3/16 10:55 AM',
      status: colorObj.delivered || ''
    },
    recieved: {
      text: 'Order received by the dispatch',
      icon: '.',
      time: '5/3/16 10:55 AM',
      status: colorObj.recieved || ''
    },
    pending: {
      text: 'waiting for pick up',
      icon: '',
      time: '5/3/16 10:55 AM',
      status: colorObj.pending || ''
    },
    accepted: {
      text: 'Accepted by chef',
      icon: '.',
      time: '5/3/16 10:55 AM',
      status: colorObj.accepted || ''
    },
    rejected: {
      text: 'Your order was rejected by chef',
      icon: '.',
      time: '5/3/16 10:55 AM',
      status: colorObj.rejected || ''
    },
    cancelled: {
      text: 'You cancelled this order',
      icon: '.',
      time: '5/3/16 10:55 AM',
      status: colorObj.cancelled || ''
    },
  };

  return (
    <Fragment>
      <div
        className={`tracking-absolute ${show ? 'tracking' : 'not-tracking'}`}
      >
        <div className="bg-color">
          <div className={'tracking-content content-bg-color'}>
            <TrackingHeader open={show} handleClick={closeTrackingDropdown} />
            <div className="tracking-content-body">
              {
                Object.keys(trackingStatus).map(dom => (
                  <Timeline
                    key={shortId.generate()}
                    icon={trackingStatus[dom].icon}
                    time={trackingStatus[dom].time}
                    status={trackingStatus[dom].text}
                    classNames={trackingStatus[dom].status}
                  />
                ))
              }
            </div>
          </div>
        </div>
      </div>
      <div className={`${show ? 'tracking-backdrop' : ''}`} />
    </Fragment>
  );
};

const mapStateToProps = ({
  displayTrackingReducer: { show, status },
}) => ({
  show,
  status,
});

export default connect(
  mapStateToProps,
  { closeTrackingDropdown: trackingDisplayClose,
  }
)(Tracking);

Tracking.propTypes = {
  closeTrackingDropdown: PropTypes.func.isRequired,
  show: PropTypes.bool.isRequired,
  status: PropTypes.string.isRequired,
};
