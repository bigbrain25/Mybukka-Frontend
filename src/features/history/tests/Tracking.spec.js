import React from 'react';
import { Provider } from 'react-redux';
import Tracking from '../components/Tracking';

const initialState = {
  displayTrackingReducer: { show: true, status: 'pending' },
};

const store = mockStore(initialState);

describe('Tracking component', () => {
  const props = {
    show: false,
    closeTrackingDropdown: jest.fn(),
    handleClick: jest.fn(),
    status: 'delivered'
  };

  const { rerender, container, getByTestId } = render(
    <Provider store={store}>
      <Tracking {...props} />
    </Provider>
  );

  it('renders properly', () => {
    expect(container).toMatchSnapshot();
  });

  it('re renders properly when rejected status', () => {
    const newProps = {
      ...props,
      show: true,
      status: 'rejected',
    };
    rerender(
      <Provider store={store}>
        <Tracking {...newProps} />
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });

  it('re renders properly when pending status', () => {
    const newProps = {
      ...props,
      show: true,
      status: 'pending',
    };
    rerender(
      <Provider store={store}>
        <Tracking {...newProps} />
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });

  it('re renders properly when accepted status', () => {
    const newProps = {
      ...props,
      show: true,
      status: 'accepted',
    };
    rerender(
      <Provider store={store}>
        <Tracking {...newProps} />
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });

  it('re renders properly when cancelled status', () => {
    const newProps = {
      ...props,
      show: true,
      status: 'cancelled',
    };
    rerender(
      <Provider store={store}>
        <Tracking {...newProps} />
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });

  it('triggers the closeTrackingDropdown function when clicked', () => {
    const button = getByTestId('button');
    fireEvent.click(button);

    expect(props.show).toBeFalsy();
  });
});
