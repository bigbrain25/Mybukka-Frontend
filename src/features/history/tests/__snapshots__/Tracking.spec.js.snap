// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tracking component re renders properly when accepted status 1`] = `
<div>
  <div
    class="tracking-absolute tracking"
  >
    <div
      class="bg-color"
    >
      <div
        class="tracking-content content-bg-color"
      >
        <div
          class="
    position-fixed
    tracking-content-header"
        >
          <div
            class="row m-0 tracking-header"
          >
            <button
              class="tracking-cancel-btn"
              data-dismiss=""
              data-target=""
              data-testid="button"
              data-toggle=""
              id="oO_HcWqMP"
              type="button"
            >
              <svg
                fill="currentColor"
                height="24"
                viewBox="0 0 24 24"
                width="24"
              >
                <defs>
                  <path
                    d="M0 1.5L1.5 0 8 6.5 14.5 0 16 1.5 9.5 8l6.5 6.5-1.5 1.5L8 9.5 1.5 16 0 14.5 6.5 8z"
                    id="icon-close_svg__a"
                  />
                </defs>
                <use
                  transform="translate(4 4)"
                  xlink:href="#icon-close_svg__a"
                />
              </svg>
            </button>
            <h5>
              Logistics details
            </h5>
          </div>
          <hr
            class="text-muted m-0 border-box col-12"
          />
        </div>
        <div
          class="tracking-content-body"
        >
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  order dispatched
                   
                  <i>
                    ✈...
                  </i>
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Order received by the dispatch
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left timeline-active"
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  waiting for pick up
                   
                  <i>
                    ✈...
                  </i>
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Accepted by chef
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Your order was rejected by chef
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  You cancelled this order
                   
                  .
                </h5>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="tracking-backdrop"
  />
</div>
`;

exports[`Tracking component re renders properly when cancelled status 1`] = `
<div>
  <div
    class="tracking-absolute tracking"
  >
    <div
      class="bg-color"
    >
      <div
        class="tracking-content content-bg-color"
      >
        <div
          class="
    position-fixed
    tracking-content-header"
        >
          <div
            class="row m-0 tracking-header"
          >
            <button
              class="tracking-cancel-btn"
              data-dismiss=""
              data-target=""
              data-testid="button"
              data-toggle=""
              id="oO_HcWqMP"
              type="button"
            >
              <svg
                fill="currentColor"
                height="24"
                viewBox="0 0 24 24"
                width="24"
              >
                <defs>
                  <path
                    d="M0 1.5L1.5 0 8 6.5 14.5 0 16 1.5 9.5 8l6.5 6.5-1.5 1.5L8 9.5 1.5 16 0 14.5 6.5 8z"
                    id="icon-close_svg__a"
                  />
                </defs>
                <use
                  transform="translate(4 4)"
                  xlink:href="#icon-close_svg__a"
                />
              </svg>
            </button>
            <h5>
              Logistics details
            </h5>
          </div>
          <hr
            class="text-muted m-0 border-box col-12"
          />
        </div>
        <div
          class="tracking-content-body"
        >
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  order dispatched
                   
                  <i>
                    ✈...
                  </i>
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Order received by the dispatch
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left timeline-active"
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  waiting for pick up
                   
                  <i>
                    ✈...
                  </i>
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Accepted by chef
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Your order was rejected by chef
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  You cancelled this order
                   
                  .
                </h5>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="tracking-backdrop"
  />
</div>
`;

exports[`Tracking component re renders properly when pending status 1`] = `
<div>
  <div
    class="tracking-absolute tracking"
  >
    <div
      class="bg-color"
    >
      <div
        class="tracking-content content-bg-color"
      >
        <div
          class="
    position-fixed
    tracking-content-header"
        >
          <div
            class="row m-0 tracking-header"
          >
            <button
              class="tracking-cancel-btn"
              data-dismiss=""
              data-target=""
              data-testid="button"
              data-toggle=""
              id="oO_HcWqMP"
              type="button"
            >
              <svg
                fill="currentColor"
                height="24"
                viewBox="0 0 24 24"
                width="24"
              >
                <defs>
                  <path
                    d="M0 1.5L1.5 0 8 6.5 14.5 0 16 1.5 9.5 8l6.5 6.5-1.5 1.5L8 9.5 1.5 16 0 14.5 6.5 8z"
                    id="icon-close_svg__a"
                  />
                </defs>
                <use
                  transform="translate(4 4)"
                  xlink:href="#icon-close_svg__a"
                />
              </svg>
            </button>
            <h5>
              Logistics details
            </h5>
          </div>
          <hr
            class="text-muted m-0 border-box col-12"
          />
        </div>
        <div
          class="tracking-content-body"
        >
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  order dispatched
                   
                  <i>
                    ✈...
                  </i>
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Order received by the dispatch
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left timeline-active"
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  waiting for pick up
                   
                  <i>
                    ✈...
                  </i>
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Accepted by chef
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Your order was rejected by chef
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  You cancelled this order
                   
                  .
                </h5>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="tracking-backdrop"
  />
</div>
`;

exports[`Tracking component re renders properly when rejected status 1`] = `
<div>
  <div
    class="tracking-absolute tracking"
  >
    <div
      class="bg-color"
    >
      <div
        class="tracking-content content-bg-color"
      >
        <div
          class="
    position-fixed
    tracking-content-header"
        >
          <div
            class="row m-0 tracking-header"
          >
            <button
              class="tracking-cancel-btn"
              data-dismiss=""
              data-target=""
              data-testid="button"
              data-toggle=""
              id="oO_HcWqMP"
              type="button"
            >
              <svg
                fill="currentColor"
                height="24"
                viewBox="0 0 24 24"
                width="24"
              >
                <defs>
                  <path
                    d="M0 1.5L1.5 0 8 6.5 14.5 0 16 1.5 9.5 8l6.5 6.5-1.5 1.5L8 9.5 1.5 16 0 14.5 6.5 8z"
                    id="icon-close_svg__a"
                  />
                </defs>
                <use
                  transform="translate(4 4)"
                  xlink:href="#icon-close_svg__a"
                />
              </svg>
            </button>
            <h5>
              Logistics details
            </h5>
          </div>
          <hr
            class="text-muted m-0 border-box col-12"
          />
        </div>
        <div
          class="tracking-content-body"
        >
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  order dispatched
                   
                  <i>
                    ✈...
                  </i>
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Order received by the dispatch
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left timeline-active"
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  waiting for pick up
                   
                  <i>
                    ✈...
                  </i>
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Accepted by chef
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Your order was rejected by chef
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  You cancelled this order
                   
                  .
                </h5>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="tracking-backdrop"
  />
</div>
`;

exports[`Tracking component renders properly 1`] = `
<div>
  <div
    class="tracking-absolute tracking"
  >
    <div
      class="bg-color"
    >
      <div
        class="tracking-content content-bg-color"
      >
        <div
          class="
    position-fixed
    tracking-content-header"
        >
          <div
            class="row m-0 tracking-header"
          >
            <button
              class="tracking-cancel-btn"
              data-dismiss=""
              data-target=""
              data-testid="button"
              data-toggle=""
              id="oO_HcWqMP"
              type="button"
            >
              <svg
                fill="currentColor"
                height="24"
                viewBox="0 0 24 24"
                width="24"
              >
                <defs>
                  <path
                    d="M0 1.5L1.5 0 8 6.5 14.5 0 16 1.5 9.5 8l6.5 6.5-1.5 1.5L8 9.5 1.5 16 0 14.5 6.5 8z"
                    id="icon-close_svg__a"
                  />
                </defs>
                <use
                  transform="translate(4 4)"
                  xlink:href="#icon-close_svg__a"
                />
              </svg>
            </button>
            <h5>
              Logistics details
            </h5>
          </div>
          <hr
            class="text-muted m-0 border-box col-12"
          />
        </div>
        <div
          class="tracking-content-body"
        >
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  order dispatched
                   
                  <i>
                    ✈...
                  </i>
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Order received by the dispatch
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left timeline-active"
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  waiting for pick up
                   
                  <i>
                    ✈...
                  </i>
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Accepted by chef
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  Your order was rejected by chef
                   
                  .
                </h5>
              </div>
            </div>
          </div>
          <div
            class="timeline"
          >
            <div
              class="timeline-container timeline-left "
            >
              <div
                class="timeline-content"
              >
                <h5
                  class="font-size-14 suggested-item-price pt-2"
                >
                  5/3/16 10:55 AM
                </h5>
                <h5
                  class="font-size-14 suggested-item-price pb-3"
                >
                  You cancelled this order
                   
                  .
                </h5>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="tracking-backdrop"
  />
</div>
`;
